Switch to graphlab-twill

erickt opened the pull request.

Commit:
whitespace cleanup

Commit:
switch over to using graphlab-twill to run graphlab

Commit:
Add graphlab-twill to travis

cglewis commented on src/main/java/org/lab41/dendrite/services/analysis/GraphLabService.java:
```diff
@@ -162,68 +166,36 @@ private void runExport(DendriteGraph graph, String jobId, Path exportDir) throws
+        String algorithmPath = config.getString("metagraph.template.graphlab.algorithm-path") + "/" + algorithm;
+
+        String[] args = new String[] {
+                "../graphlab-twill/bin/graphlab-twill",
```
making this an option in the properties file would be useful

cglewis commented on src/main/java/org/lab41/dendrite/services/analysis/GraphLabService.java:
```diff
@@ -162,68 +166,36 @@ private void runExport(DendriteGraph graph, String jobId, Path exportDir) throws
+        String[] args = new String[] {
+                "../graphlab-twill/bin/graphlab-twill",
+                "-i", clusterSize,
+                "localhost:2181",
```
making this an option in the properties file would be useful

Commit:
turn path to graphlab-twill into a property

also removes "metagraph.template" from the config path. This property
is not yet being versioned in the metagraph, so it doesn't make sense
to have this reference in the property name.

Commit:
Log the arguments and exit code of running graphlab-twill

cglewis commented on src/main/java/org/lab41/dendrite/services/analysis/GraphLabService.java:
```diff
@@ -162,68 +167,41 @@ private void runExport(DendriteGraph graph, String jobId, Path exportDir) throws
+        List<String> args = Lists.newArrayList(
+                graphlabTwillPath,
+                "-i", clusterSize,
+                "localhost:2181",
```
this should be a property, at least for prod.

Commit:
Merge remote-tracking branch 'remotes/origin/master' into twill

Commit:
Merge remote-tracking branch 'remotes/origin/master' into twill

Commit:
convert twill zookeeper url into a property

Commit:
Make sure the graphlab temp directory is world writable

Commit:
fix a typo in the algorithm property

cglewis commented:
No longer necessary as #157 pulls in these changes.
